media/css/firefox/privacy/common.scss

```diff
@@ -24,8 +24,9 @@ $brand-theme: 'firefox';
 
 /* -------------------------------------------------------------------------- */
 // Reduce vertical padding on stacked split components
+// Avoid gap space between top split component and nav, issue #11457
 
-.privacy-promise-hero,
```

Comment:
this used to only apply the fix to the `/privacy` page, new selector should work across all privacy pages where the first child of the main element is a `nospace` component